src/Core/Bases/A11yElement.cs

```diff
@@ -417,7 +419,7 @@ public T GetPlatformPropertyValue<T>(int propertyId)
         {
             var property = this.PlatformProperties?.ById(propertyId);
             if (property == null) return default;
-            if (!(property.Value is T)) throw new AxeWindowsException(Invariant($"Expected property.Value, which is type {property.Value.GetType().Name}, to be type {typeof(T).Name}"));
+            if (!(property.Value is T)) throw new AxeWindowsException(ExtensionMethods.WithParameters(ErrorMessages.PropertyValueTypeUnexpected, property.Value.GetType().Name, typeof(T).Name));
```

Comment:
There are 7 places where you're using `ExtensionMethods.WithParameters(foo...` instead of `foo.WithParameters(...`

I'm not sure why the 7 instances are different...

Reply:
Those instances can't use the normal syntax because they're dynamic objects, see: https://riptutorial.com/csharp/example/3775/extension-methods-aren-t-supported-by-dynamic-code-